riff-raff/app/persistence/ContinuousDeploymentConfigRepository.scala

```diff
@@ -0,0 +1,40 @@
+package persistence
+
+import java.util.UUID
+
+import cats.data.Validated
+import ci.{ContinuousDeploymentConfig, Trigger}
+import com.gu.scanamo.{DynamoFormat, Scanamo, TypeCoercionError}
+import com.gu.scanamo.syntax._
+import conf.Configuration
+import org.joda.time.DateTime
+
+object ContinuousDeploymentConfigRepository {
+  val tableName = "continuous-deployment-config"
+  val client = Configuration.continuousDeployment.dynamoClient
```

Comment:
I'm not a huge fan of doing IO from an Object constructor unless that Object is the 'edge' of the program (where it can't be helped). Better to take this as an arg so the dirty edge can do the IO of fetching it?